### ISSUE TITLE
Use file names for assets with Expo Assets

### ISSUE THREAD

**Issue by jbaxleyiii** [bndkt/react-native-app-clip#49]
Is it possible to use the filename approach of [expo-assets](https://docs.expo.dev/versions/latest/sdk/asset/) (i.e. source="my-image" instead of souce={require("../my-image.png")}) with app clips? So far I'm not finding it to work but wondering if I'm missing a setting or something?

**Comment by nathan-ahn** [comment]
I'm not too familiar with the filename approach of `expo-assets`, and I can't seem to find an example in the linked docs of its usage. Is this using a `"my-image"` which is located at the root of the project? 

Additionally, are you having these issues with App Clip on a dev or release build?

**Comment by jbaxleyiii** [comment]
@nathan-ahn yes! It uses [hybrid resources](https://reactnative.dev/docs/images#images-from-hybrid-apps-resources) but maybe it is more of pain than it is worth. Right now just in dev. If I move to using an import statement the issue goes away so I suspect that is the easiest path
